tst/EnergyPlus/unit/DXCoils.unit.cc

```diff
@@ -3579,6 +3579,24 @@ TEST_F(EnergyPlusFixture, TestMultiSpeedCoilsAutoSizingOutput)
     // Design Capacity at speed 2 and speed 1
     EXPECT_NEAR(32731.91, state->dataDXCoils->DXCoil(1).MSRatedTotCap(2), 0.01);
     EXPECT_NEAR(16365.95, state->dataDXCoils->DXCoil(1).MSRatedTotCap(1), 0.01);
+    // Check EIO reporting
```

Comment:
@lymereJ OK, I just pushed a commit to this branch which uses raw string literals for comparing chunks of text rather than the delimited string approach.  I came across one "gotcha", and I'm going to tag a few other people who write a lot of unit tests so they see what is going on.

@rraustad @mjwitte @Nigusse 

In a unit test, if you want to compare a multiline bit of text, such as an EIO section, raw string literals are nice:

```c++
std::string const eio_text = R"EIO(First Line
Second Line
Third Line
)EIO";
EXPECT_TRUE(compare_eio_stream(eio_text));
```

This makes it very easy to paste in a chunk of EIO text without having to worry about the delimited string calls and breaking it up into individual quoted lines.

However, there is one gotcha, which is if the section of raw text has a trailing string, as this example does after the 16.03 on the next to last line.  Most editors will try to clean off trailing spaces from lines, which is problematic, because now this chunk of raw text won't exactly match.  We could have a discussion about whether the EIO should have trailing spaces, but that's for a different day.  For now, you'll notice it doesn't actually have trailing spaces -- it has pipe symbols "||".  This rarely used character is used as a placeholder for the trailing spaces so that the editor won't chop them off.  I then added a tiny worker function in the unit test fixture to replace them at runtime with spaces, so that the unit test matches.  So now it is:

```c++
std::string const eio_text = R"EIO(First Line
Second Line with 3 trailing spaces|||
Third Line with 1 trailing space|
)EIO";
replace_pipes_with_spaces(eio_text);
EXPECT_TRUE(compare_eio_stream(eio_text));
```

Reply:
I see that and was also wondering why there are trailing spaces. I see those in Format_991 and Format_995. This is a good work around for now.